clang/include/clang/Driver/Job.h

```diff
@@ -194,15 +200,23 @@ class Command {
   virtual int Execute(ArrayRef<Optional<StringRef>> Redirects,
                       std::string *ErrMsg, bool *ExecutionFailed) const;
 
-  /// Store a custom driver diagnostic message upon a particular error code
-  /// returned by the command
+  /// Store a custom driver diagnostic message and if the compilation should
+  /// exit upon a particular error code returned by the command
   void addDiagForErrorCode(int ErrorCode, StringRef CustomDiag);
```

Comment:
This method implementation has not changed. I think the comment should be left as-is.
```suggestion
  /// Store a custom driver diagnostic message upon a particular error code
  /// returned by the command
  void addDiagForErrorCode(int ErrorCode, StringRef CustomDiag);
```